server/apps/users/urls.py

```diff
@@ -10,4 +10,7 @@
     path('refresh-access-token/', views.RefreshTokenView.as_view()),
     # to'do app views
     path('todo-list/', views.TodoListView.as_view()),
+    path('create-todo/', views.CreateTodoView.as_view()),
+    path('update-todo/', views.UpdateTodoView.as_view()),
+    path('delete-todo/<int:pk>/', views.DeleteTodoView.as_view()),
 ]
```

Comment:
Your Rest standards here.

You only require to endpoints entries.

/todos - 
/todos/<int:pk>

Reply:
Implement in latest commit